py/desispec/io/meta.py

```diff
@@ -239,6 +239,7 @@ def findfile(filetype, night=None, expid=None, camera=None,
         fibermap = '{specprod_dir}/preproc/{night}/{expid:08d}/fibermap-{expid:08d}.fits',
         preproc = '{specprod_dir}/preproc/{night}/{expid:08d}/preproc-{camera}-{expid:08d}.fits{compsuffix}',
         preproc_for_cte = '{specprod_dir}/preproc/{night}/{expid:08d}/ctepreproc-{camera}-{expid:08d}.fits{compsuffix}',
+        preproc_for_dark = '{specprod_dir}/dark_preproc/{night}/{expid:08d}/preproc-{camera}-{expid:08d}.fits{compsuffix}',
```

Comment:
Good that these are kept in a different directory tree since they are different from normal preproc images (dark hasn't been applied).  In addition to changing the directory name, let's also change the filename prefix (preprocdark?) to follow the principle that we shouldn't have two files with the same name in two different directories (i.e. the filename alone is enough to disambiguate)

Reply:
done